### ISSUE TITLE
Unable to get VVV up and running

### ISSUE THREAD

**Issue by TanKarWoon91** [Varying-Vagrant-Vagrants/VVV#2263]
## What was The Command Used To Provision

```shell
command goes here
```

## What Kind of VVV Provision Was This

This was a fresh install
This was an attempt to upgrade to a new version of VVV
This was a reprovision of an already working VVV

## Logs/What Broke

```yaml
config/config.yml goes here
```

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

**Comment by Mte90** [comment]
Closing as the ticket is not filled with any information.

**Comment by TanKarWoon91** [comment]
![image](https://user-images.githubusercontent.com/73118800/96459108-4d65c480-1254-11eb-92c6-4e66d1d16ad0.png)

**Comment by TanKarWoon91** [comment]
should i uninstall host updater in that sense??

**Comment by TanKarWoon91** [comment]
I tried to uninstall hostupdater and reinstall but it is not working

**Comment by Mte90** [comment]
No, the issue is that require more privileges if you see at the bottom of your screenshot.
There is also a link to https://github.com/goodhosts/vagrant#windows-uac-prompt

Basically the plugin that change the hosts file requires permission to edit and you are not giving it.

**Comment by tomjn** [comment]
TLDR: You didn't run this in an elevated prompt, you must use an elevated prompt as the instructions said.

https://www.howtogeek.com/194041/how-to-open-the-command-prompt-as-administrator-in-windows-8.1/

**Comment by TanKarWoon91** [comment]
tomjin i got the administrator command prompt window open already, and tried to run the command "vagrant up", but it still does not work....
 default: [vagrant-goodhosts] Warning: The vagrant-hostsupdater plugin is installed, hostsupdater always adds the VM name even if the VagrantFile says not to. This shouldn't cause issues, but if it does, uninstall hostsupdater
How do i solve above problem

**Comment by Mte90** [comment]
This is another warning and not the error that you have. IF you don't want to see anymore that error just uninstall vagrant-hostsupdater plugin.

About your error the screen report that you don't have permission to change that file so we need to work on that.
Can you explain better what you did there?

**Comment by TanKarWoon91** [comment]
so far I got the adminstrator admin prompt running in the background, and I tried the "vagrant up" command , it is still showing the same errors

**Comment by Mte90** [comment]
What you did with that administrator prompt?
We don't have any windows tester or contributor right now so we cannot test it easily.

**Comment by TanKarWoon91** [comment]
![image](https://user-images.githubusercontent.com/73118800/96462949-c2d39400-1258-11eb-80a9-7dc35cbcf53e.png)

**Comment by TanKarWoon91** [comment]
May I know how do I run "Vagrant Up" under admin rights with the administrator prompt window?

**Comment by Mte90** [comment]
Well this is not the right way to use it.
You just copy and pasted the warning, that is not a command at all so this doesn't work.

You need to execute `vagrant up` from this terminal

**Comment by TanKarWoon91** [comment]
![image](https://user-images.githubusercontent.com/73118800/96463450-50af7f00-1259-11eb-99b2-ac2393ec6177.png)
What do i need to do next?

**Comment by TanKarWoon91** [comment]
But have another error message:
==> default: [vagrant-goodhosts] Issue executing goodhosts CLI: "cl" is an invalid IP address

**Comment by Mte90** [comment]
## Recap

Vagrant need to be executed from the folder where it is VVV.  
From the administrator prompt you are executing inside `system32` folder and this is wrong as Vagrant report that cannot find anything.
So you need to `cd` the folder where it is VVV with administrator prompt.

Another point is the command that you executed from the plugin readme, this let you to get permissions on the file and not using the administrator prompt at all, if you execute that command as described in the readme.
So as you did everything is fine now.

So the last screenshot, report an error that I think it is on your `config.yml` file that you can share?

**Comment by TanKarWoon91** [comment]
![image](https://user-images.githubusercontent.com/73118800/96467132-34154600-125d-11eb-824c-757442e91adf.png)
![image](https://user-images.githubusercontent.com/73118800/96467205-42fbf880-125d-11eb-88f8-207e2510384a.png)
![image](https://user-images.githubusercontent.com/73118800/96467246-4db68d80-125d-11eb-9036-86b7e83373b9.png)
![image](https://user-images.githubusercontent.com/73118800/96467297-5a3ae600-125d-11eb-99ac-10f619aa27ad.png)
![image](https://user-images.githubusercontent.com/73118800/96467329-6161f400-125d-11eb-96eb-ab499f7348c9.png)

**Comment by Mte90** [comment]
It was enough copy and paste here without any screenshot but this is the `default-config.yml` and not `config.yml`.
Did you check our documentation? https://varyingvagrantvagrants.org/docs/en-US/installation/

**Comment by TanKarWoon91** [comment]
Here is the config yml file

**Comment by Mte90** [comment]
Ehm there isn't the config file...

**Comment by Mte90** [comment]
We released a new version of the `vagrant-goodhosts` plugin that should fix your issues.
You need to update it and everything should work now.

**Comment by tomjn** [comment]
`vagrant plugin update` will update it, then run `vagrant up --provision` in the VVV folder

**Comment by TanKarWoon91** [comment]
I typed in the below to install goodhost
"git clone https://github.com/goodhosts/vagrant vagrant-goodhosts
cd vagrant-goodhosts
git checkout develop
./package.shup
vagrant plugin install vagrant-goodhosts-*.gem"
and following that i typed "vagrant plugin update"
and "vagrant up provision" after that

**Comment by TanKarWoon91** [comment]
![image](https://user-images.githubusercontent.com/73118800/96540736-0cf85c00-12d1-11eb-83e8-d179e111370a.png)
This is the comments i got after i typed in the commands mentioned above

**Comment by tomjn** [comment]
@TanKarWoon91 you only had to run `vagrant plugin update` to update goodhosts
